README.md

```diff
@@ -1,8 +1,58 @@
-# Open Container - Container Distributable Image Specification
+# Open Container - Distributable Image Specification
 
-[Open Container Initiative](http://www.opencontainers.org/) Specifications for standards on Operating System process and application containers.
+The [Open Container Initiative](http://www.opencontainers.org/) develops specifications for standards on Operating System process and application containers.
```

Comment:
Ideally there will be a symmetrical change of lines 1-5 here on opencontainers/runtime-spec's README, /cc @vbatts